multicluster/cmd/multicluster-controller/options.go

```diff
@@ -32,6 +32,8 @@ type Options struct {
 	options        ctrl.Options
 	// The Service ClusterIP range used in the member cluster.
 	ServiceCIDR string
+	// PodCIDRs is the Pod IP address CIDRs.
```

Comment:
PodCIDRs is the Pod IP address CIDRs of the member cluster.

Reply:
Fixed.